CONTRIBUTING.md

```diff
@@ -86,11 +86,11 @@ IGNORE IMAGE ANALYSIS
 ```
 
 | Rule | Description | Notes / Examples |
-|-|-|-|
-| `INVERT` | Inverts specified elements. | **Dynamic Mode**: INVERT only for dark images that are invisible on dark backgrounds. |
-| `CSS` | Adds custom CSS to a web page. | `!important` keyword should be specified for each CSS property to prevent overrides by other stylesheets.<br>**Dynamic mode** supports `${COLOR}` template, where `COLOR` is a color value before the inversion. <br>*Example*: `${white}` will become `${black}` in dark mode. |
-| `IGNORE INLINE STYLE` | Prevents inline style analysis of matched elements. | *Example*: `<p style="color: red">` element's style attribute will not be changed. |
-| `IGNORE IMAGE ANALYSIS` | Prevents background images from being analyzed for matched selectors. |  |
+|---|---|---|
+| **INVERT** | Inverts specified elements. | **Dynamic Mode**: INVERT only for dark images that are invisible on dark backgrounds. |
```

Comment:
Why making it bold?

Reply:
Just to make it stand out a little more.  Because of the `&nbsp;` characters, these can't use the backtick (code) style, leaving bold, italics, and font-size as the only real choices for emphasis.  I felt that bold was the best choice.

Not sure where you are in your career, but you might benefit from being mindful not to wander too far into [this realm](https://www.thefreedictionary.com/micromanage). :)